Azure/azure-cli
# [App Service] az logicapp config appsettings [set/delete/list]: Add new commands for logicapp settings

**Description**

Add `az logicapp config appsettings [set/delete/list]` commands.

**Testing Guide**

First create a logicapp.

`az logicapp config appsettings set -n {} --settings "mysetting=myvalue"` to update settings.
`az logicapp config appsettings delete -n {}--setting-names mysettingname` to delete settings.
`az logicapp config appsettings list -n {}` to list settings.
---

This checklist is used to make sure that common guidelines for a pull request are followed.
